include/boost/decimal/detail/cmath/tgamma.hpp

```diff
@@ -17,6 +17,62 @@
 namespace boost {
 namespace decimal {
 
+namespace detail {
+
+template <BOOST_DECIMAL_DECIMAL_FLOATING_TYPE T,
+          const std::size_t CoefsSize>
+std::array<T, CoefsSize> coefs_array_table = {{ T { } }};
```

Comment:
Should this be `static constexpr`?

Reply:
> Should this be `static` `constexpr`

Yes. I just saw that on the secong go-around.

An interesting thing did, however, happen when generating a list of $44$ coefficients for the $128$-bit case. The number of `constexpr` iterations exceeded the default allowed value of around a million for some compilers. Now all these compilers have ways to set the maximum number of allowed `constexpr` iterations, but I usually draw the line at the million.

What caught my eye was _where_ the iterations went. lots of shifting and I'm startnig to really want more and more to rework long division for big ints in this project...